gitlens/gitlens-account.md

```diff
@@ -9,6 +9,14 @@ taxonomy:
 
 Whether a solo user on a personal license, or an owner managing multiple teams across organizations, learn how to manage your GitKraken GitLens account(s), subscription(s), and organization(s).
 
+The first question to answer: do I need an account to use GitLens or GitLens+? 
+
++ GitLens is COMPLETELY FREE, no account is required, and can be used on any local repository.
```

Comment:
Although I understand this is for emphasis, I think the use of capital words in these docs (and the other page) are a bit out of place. If we did want to emphasize these words, maybe bold is a better option?

Reply:
Changed to be **bold** instead